package.json

```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
+    "assign": "git://github.com/newoga/assign.git",
```

Comment:
Yeah originally I wanted to design the plugin to create a file with the right implementation in the build output, but I couldn't figure out how to do it and I didn't see an easy way to resolve relative paths for components in babel, so making it a package with a static import name was easiest.